fv3core/pace/fv3core/stencils/c_sw.py

```diff
@@ -703,13 +727,15 @@ def __call__(
         )
 
         # TODO(eddied): We pass the same fields 2x to avoid GTC validation errors
```

Comment:
This actually might create issues. We should track every aliasing and put them out

Reply:
Should I make an issue for this? Or do you want to dive into it here?

Reply:
done